tensorrt_llm/_torch/auto_deploy/models/quant_config_reader.py

```diff
@@ -78,6 +78,8 @@ def has(cls, reader_cls: str) -> bool:
 
 @QuantConfigReaderRegistry.register("modelopt")
 class ModelOPTQuantConfigReader(QuantConfigReader):
+    _ALWAYS_EXCLUDE = ("lm_head", "model.embed_tokens")
```

Comment:
seriously lol? we need to hard-code those exceptions into the inference framework itself? 😅